tests/rip7560/test_rip7712.py

```diff
@@ -0,0 +1,58 @@
+from eth_abi.packed import encode_packed
+
+from tests.types import RPCErrorCode
+from tests.utils import send_bundle_now, assert_rpc_error
+
+
+def get_nonce(w3, nonce_manager, address, key):
+    get_nonce_call_data = encode_packed(
+        ["address", "uint192"],
+        [address, key]
+    )
+    return w3.eth.call(
+        {"to": nonce_manager.address, "data": get_nonce_call_data}
+    )
+
+
+def pack_nonce(key, value):
+    return encode_packed(
+        ["uint192", "uint64"],
+        [key, value]
+    )
+
+
+def test_eth_sendTransaction7560_7712_valid(w3, wallet_contract, nonce_manager, tx_7560):
```

Comment:
tip: parameterize with multiple "key" value, including zero.

Reply:
Zero `nonceKey` behaves differently and is checked by every other test. 
Other values are all same, so I don't see a big reason to parametrize.